From 3bf6f622742953fed0a381e55d07f0a8ab2dd97b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 06 May 2019 14:48:09 +0300
Subject: [PATCH] [SW-0] pll: include stdbool.h for booleans

pll.c uses bool type and values, but does not include the proper
header.
Include stdbool.h.

Change-Id: I27d1f73f3d9c45c678f64b4ba534f4e77fe9ae85
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index 2c2409a..b365c52 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -5,6 +5,8 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
+#include <stdbool.h>
+
 #include "preboot.h"
 #include "uart.h"
 #include "pll.h"
